zooma-mappings-wf/rules/common.smk

```diff
@@ -86,3 +86,13 @@ def get_split_zooma_mapping_report_inputs(accessions, lp):
     for acc in accessions:
         inputs.append(f"{lp}/{acc}/{acc}-apply_fixes_zooma_mappings.done")
     return inputs
+
+def get_mem_mb(wildcards, attempt):
+    """
+    To adjust resources in the rules 
+    attemps = reiterations + 1
+    Max number attemps = 4
```

Comment:
Where is this max attempt for Snakemake specified?

Reply:
The parameter `attempt` allows us to adjust resources based on how often the job has been restarted (controlled by option `--retries`).  When Snakemake is executed with `--retries 3`, it will try to restart a failed job 3 times before it gives up. 

The number of retries is specified in a script upstream (in the snakemake call).

Reply:
Yes that was my concern that if config changed to do more than 4 attempts, it will fail.